### PR TITLE
Merge in pre connect-es rpc/js

### PR DESCRIPTION
**Pull request by edaniels**
Discussed with @mcous and @ethanlook, we're going to merge the rpc/js into this repo before the connect-es changes in order to ease the transition as far as build systems go. Later PRs will bring the connect-es code in and then later, the examples and gh actions.

Lint disables you see applied here are done because it's not worth fixing code that is about to be replaced. It works as is.

### CODE REVIEW

File: src/app/viam-transport.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { grpc } from '@improbable-eng/grpc-web';
-import { dialDirect } from '@viamrobotics/rpc';
+import { dialDirect } from '../rpc/dial';
```
Comment (ethanlook) [inline src/app/viam-transport.ts:2]: What do you think about adding an `rpc/index.ts` to simplify imports?

**Review by ethanlook — APPROVED** [review]
Awesome, thanks! This definitely will reduce the burden of rpc changes in the TS SDK for the time being.

File: src/app/viam-transport.ts (hunk 1 above)
Comment (edaniels) [reply src/app/viam-transport.ts:2]: sure!

**Review by njooma — APPROVED** [review]
I'm assuming the `rpc` directory is basically just copied from goutils and will trust that it functions the same

**Pull request merged**